tools/jsdoc/plugins/hifiJSONExport.js

```diff
@@ -1,8 +1,22 @@
 exports.handlers = {
     processingComplete: function(e) {
         const pathTools = require('path');
-        var outputFolder = pathTools.join(__dirname, '../out');
+        var outputFolder;
         var doclets = e.doclets.map(doclet => Object.assign({}, doclet));
+
+        var argv = process.argv;
+        for(var i=0;i<argv.length;i++) {
+            if ( argv[i] == '-d' && i <= argv.length+1) {
```

Comment:
```suggestion
        for (var i=0; i < argv.length; i++) {
            if (argv[i] === '-d') {
```
` i <= argv.length + 1` will always be `true`.